### PR TITLE
[enhancement](cloud) add bvar to monitor s3 throughput&QPS

### PR DESCRIPTION
**Pull request by freemandealer**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by freemandealer** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

File: be/src/io/fs/s3_file_reader.cpp [hunk 1]
```diff
@@ -43,6 +44,9 @@ bvar::Adder<uint64_t> s3_file_reader_read_counter("s3_file_reader", "read_at");
 bvar::Adder<uint64_t> s3_file_reader_total("s3_file_reader", "total_num");
 bvar::Adder<uint64_t> s3_bytes_read_total("s3_file_reader", "bytes_read");
 bvar::Adder<uint64_t> s3_file_being_read("s3_file_reader", "file_being_read");
+bvar::LatencyRecorder s3_bytes_per_read("s3_file_reader", "bytes_per_read"); // also QPS
```
Comment (gavinchou) [inline be/src/io/fs/s3_file_reader.cpp:47]: It seems calculating the rate of s3_bytes_read_total (throughput) and s3_file_reader_read_counter (qps)
will do the work.
Adding redundant metrics is a burden for the scraping procedure (Prometheus).

Comment (freemandealer) [reply be/src/io/fs/s3_file_reader.cpp:47]: bvar has fine granularity, so it is easier to identify throughput issue (whether caused by long wait or low instantaneous speed). So I insist, but we can delete them when the optimization work is done.

**Comment by freemandealer** [comment]
run buildall

**Review by gavinchou — APPROVED** [review]

**Review by ByteYue — APPROVED** [review]

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**